Mention tools in README (#4065)
rizinorg/rizin: README.md

WASM (WebAssembly binary), Commodore VICE emulator, QNX,
Game Boy (Advance), Nintendo DS ROMs and Nintendo 3DS FIRMs.

## Tools

> Apart from the main tool - `rizin` there are also other tools tailored for specific purpose and 

Reviewer: ```suggestion
Apart from the main tool `rizin`, there are also other tools tailored for specific purposes and 
```

> useful for shell scripting or as a separate standalone tools:

Reviewer: ```suggestion
useful for shell scripting or as separate standalone tools:
```

- `rz-bin` - provides all kind of information about binary formats
- `rz-asm` - a command-line assembler and disassemblers
- `rz-diff` - a tool to compare two binaries as raw data or analyzed executables

> - `rz-hash` - allows to calculate different hashes or even encrypt the Data

Reviewer: ```suggestion
- `rz-hash` - allows to calculate different hashes or even encrypt data
```

- `rz-gg` - a small "eggs" code generator useful for exploitation purposes
- `rz-find` - binary analog of `find` tool, allowing to search patterns and bit masks

> - `rz-sign` - tool to work with FLIRT signatures

Reviewer 2: ```suggestion
- `rz-sign` - tool to create, convert and parse FLIRT signatures
```
